lib/ReactCrop.js

```diff
@@ -543,6 +559,13 @@ class ReactCrop extends Component {
 	arrayToPercent(arr, delimeter = ' ') {
 		return arr.map(number => number + '%').join(delimeter);
 	}
+	
+	preventDefault(e) {
```

Comment:
I saw this practice done in other code bases. Added it while trying to figure out IOS bugs. Feel free to not include it.